templates/httpd_pub.erb

```diff
@@ -3,5 +3,10 @@ alias /pub /var/www/html/pub
 <Location /pub>
   PassengerEnabled off
   Options +FollowSymLinks +Indexes
+<%- if scope.function_versioncmp([@apache_version, '2.4']) >= 0 -%>
```

Comment:
You will need to set this up -- for example: https://github.com/Katello/puppet-katello/blob/c12447f1c01ba837d5ddbe1aeb908e6d18e6ab57/manifests/config.pp#L4